framework/DataObjects/XHistorySet.py

```diff
@@ -228,6 +238,15 @@ def _toCSV(self,fname,start=0,**kwargs):
     ## obtain slices to write subset CSVs
     ordered = list(o for o in self.getVars('output') if o in keep)
     for i in range(len(data[self.sampleTag].values)):
-      rlz = data.isel(**{self.sampleTag:i})[ordered].dropna(self.indexes[0])
+      #########################################################################
+      ## Is it correct?
+      ## why you were dropping the index? I think it is logical I want to have the "time" in the output of the HistorySet
+      ##TODO:
+      ## Conceptually we always considered the "PivotParameter" an output. So If I request "output" in <what> or I do not specify <what> I should
+      ## always add the pivotParameter value in the printing.
+      ## Please check
+      ## rlz = data.isel(**{self.sampleTag:i})[ordered] (AA: actually this does not change anything)
+      rlz = data.isel(**{self.sampleTag:i})[ordered].dropna(self.indexes[0]) #
```

Comment:
I don't think the intent was to drop the pivotParameter; on the contrary, the intent is to drop any pivot values that have NaN values, right?  Is this comment still relevant?

Reply:
No, I got it later. The comment is not valid anymore and I am removing it